flake8_trio/runner.py

```diff
@@ -89,10 +91,5 @@ def visit(self, node: ast.AST):
         self.visitors.update(novisit)
 
         # restore any outer state that was saved in the visitor method
-        for subclass in self.visitors:
+        for subclass in *self.utility_visitors, *self.visitors:
```

Comment:
I think this is an implicit tuple?  Would feel safer with parens around it though,

Reply:
yep. I usually default to not putting explicit tuple parentheses unless necessary, but would maybe be nice to have them here.